lib/global_registry_bindings/entity/push_entity_methods.rb

```diff
@@ -72,6 +69,16 @@ def dig_global_registry_id_from_entity(entity, type, parent_type = nil)
             item['client_integration_id'] == id.to_s
           end&.dig('id')
         end
+
+        def ensure_parent_entity_has_global_registry_id!
```

Comment:
nope, does there need to be? My thinking was bang method would raise exceptions.